test/unit/locators/test_windowmanager.py

```diff
@@ -1,312 +1,240 @@
 import unittest
 import uuid
 
-from mockito import mock, unstub
+from mockito import mock
 
-from SeleniumLibrary.locators.windowmanager import WindowManager
+from SeleniumLibrary.locators import WindowManager
 
 
 class WindowManagerTests(unittest.TestCase):
 
     def test_select_with_invalid_prefix(self):
-        manager = WindowManager()
-        browser = mock()
+        manager = WindowManagerWithMockBrowser()
         with self.assertRaises(ValueError) as err:
-            manager.select(browser, "something=test1")
+            manager.select("something=test1")
             self.assertEqual(
                 str(err),
                 "Window locator with prefix 'something' is not supported"
             )
-        unstub()
```

Comment:
It might be good idea to call unstub() in `def tearDown`.

Reply:
Is it needed when we don't actually stub anything here? We just mock `ctx`.

Reply:
I did read the mockito docs again and you are right is it needed when doing stubs.